src/View/Helper/FormHelper.php

```diff
@@ -1850,7 +1856,7 @@ public function submit($caption = null, array $options = [])
 
         $input = $this->formatTemplate('inputSubmit', [
             'type' => $type,
-            'attrs' => $this->templater()->formatAttributes($options),
+            'attrs' => $this->templater()->formatAttributes($options, ['label']),
```

Comment:
Because it was failing a test `testFormInputSubmit` due to the default method call in _getInput. I forgot to remove it, apologies!